packages/gp2-components/src/organisms/ProjectSummaryFooter.tsx

```diff
@@ -24,15 +24,17 @@ const ProjectSummaryFooter: React.FC<ProjectSummaryFooterProps> = ({
     </IconWithLabel>
     <IconWithLabel icon={dateIcon}>
       <span>
-        {`${format(new Date(startDate), 'MMM yyyy')}${
-          endDate ? ` - ${format(new Date(endDate), 'MMM yyyy')} · ` : ''
-        }`}
-        {endDate && (
-          <span css={{ color: colors.neutral800.rgba }}>
-            {`(${formatDistance(new Date(startDate), new Date(endDate), {
-              unit: 'month',
-            })})`}
-          </span>
+        {startDate ? `${format(new Date(startDate), 'MMM yyyy')}` : 'TBD'}
+        {startDate && endDate && (
+          <>
+            {' - '}
+            {format(new Date(endDate), 'MMM yyyy')} ·{' '}
+            <span css={{ color: colors.neutral800.rgba }}>
+              {`(${formatDistance(new Date(startDate), new Date(endDate), {
+                unit: 'month',
+              })})`}
+            </span>
+          </>
         )}
```

Comment:
This is the reason I suggested a ternary here: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx